cli/src/api/templates/js-binding.ts

```diff
@@ -253,5 +253,6 @@ if (!nativeBinding) {
   throw new Error(\`Failed to load native binding\`)
 }
 
+module.exports = nativeBinding
```

Comment:
you should generate codes like this: https://github.com/napi-rs/napi-rs/blob/v2/cli/src/build.ts#L876

```js
const { foo, bar }  = nativeBinding
module.exports.foo = foo
module.exports.bar = bar
```

so that users can use `import { foo } from '@napi-rs/binding'` in esm projects

Reply:
any testcase? does `entries.forEach((k, v) => { module.exports[k] = v })` help?